components/PatientPrescriptions.tsx

```diff
@@ -0,0 +1,109 @@
+import theme from '../styles/theme'
+import {
+  Box,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Button,
+  Tabs,
+  Tab
+} from 'grommet'
+import { PrescriptionAndLocation } from '../types/types'
+
+const PatientPrescriptions = ({
+  prescriptions,
+  previousPrescriptions,
+  setQrCode,
+  setShowQrCode
+}: {
+  prescriptions: PrescriptionAndLocation[]
+  previousPrescriptions: PrescriptionAndLocation[]
+  setQrCode: any
```

Comment:
nit pick but you can enforce function definition here
```suggestion
  setQrCode: (pickupCode: string) => void
```